handleData.js

```diff
@@ -119,11 +127,52 @@ var Character = function(realName, gameName, level, gameClass, race) {
     this.characterLevel = level;
     this.characterClass = gameClass;
     this.characterRace = race;
+    this.inventory = [];
+    this.money = { "Copper Pieces"      : 0,
+                   "Silver Pieces"      : 0,
+                   "Gold Pieces"        : 0,
+                   "Platinum Pieces"    : 0,
+                   "Electrum Pieces"    : 0};
```

Comment:
Also, Pieces in the name is a bit redundant, It could just be "Copper", "Silver", "Gold", "Platinum", "Electrum"